internal/cli/up.go

```diff
@@ -40,12 +40,12 @@ var webHost = DefaultWebHost
 var webDevPort = 0
 var noBrowser bool = false
 var logActionsFlag bool = false
+var fileName string = tiltfile.FileName
```

Comment:
it would be better for command-specific flags to be on a command-specific struct, and not share each other's global state. it's pretty common to eventually want to move these commands to separate packages, cf kubectl